.rubocop.yml

```diff
@@ -31,7 +31,7 @@ Lint/AmbiguousBlockAssociation:
 Metrics/AbcSize:
   # The ABC size is a calculated magnitude, so this number can be an Integer or
   # a Float.
-  Max: 15
+  Max: 17
```

Comment:
why 17? like did you get the number from somewhere?

Reply:
Hmm, I thought we have changed this numer in the rails-api-base to 17 but I just checked and didnt find anything

Reply:
In fact we did that last year, Ownable has 17 as an example